### ISSUE TITLE
[Request device support] - Develco Motion Sensor Mini

### ISSUE THREAD

**Issue by Hagenisse** [dresden-elektronik/deconz-rest-plugin#1294]
I'd like support for this device in deCONZ, if possible?
In Phoscon GW, when trying to connect the sensor as type "Other", it says "Failed to connect". It appears in deCONZ, but only with one node cirle:

![image](https://user-images.githubusercontent.com/47867495/53287771-59e50180-3781-11e9-829a-3d2669da71c4.png)

I guess that means I won't be able to provide any cluster info?

The node info looks like this:
![image](https://user-images.githubusercontent.com/47867495/53287780-797c2a00-3781-11e9-9600-0930cb359c2b.png)

Product details from the manufacturer:
https://www.develcoproducts.com/products/sensors-and-alarms/motion-sensor-mini/

**Comment by Ronoon** [comment]
delete the device in DeConz and then reconnect it to deconz to you can follow the procedure as written in 

https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Request-Device-Support

Without  it it will mostly not be possible to give you the requestet support
Since the device has the HA1.2 profile you should be able to fullfill the answers to the requestet device support sheet

**Comment by ebaauw** [comment]
Or try and select _Read Simple Descriptor(s)_ from the left circle, while pressing the dimmer's buttons, to make sure it is awake.  When the node blinks blue, deCONZ sends the request.  If it blinks red afterwards, the device could not be reached, probably because it's asleep with the radio receiver off.

**Comment by Hagenisse** [comment]
Thanks for your replies!
And I'm sorry, I somehow managed to upload the node info for the wrong device! (A working dimmer) 

Please, let me try again. This is a motion sensor. After deleting it from deCONZ, opening the network (permit join), resetting the sensor, keeping it awake and let deCONZ find it (did not use Phoscon this time), I ended up with both node circles:
![image](https://user-images.githubusercontent.com/47867495/53299186-6d52a400-3837-11e9-8918-360bf26c6d64.png)

Here is the CORRECT node info for this sensor:
![image](https://user-images.githubusercontent.com/47867495/53299227-c8849680-3837-11e9-8cdf-4c8fc629395f.png)


Also, I tried ebaauw's suggestion and hit the _Read Simple Descriptor(s)_ a few times. Wich led to this info on the right node circle:
![image](https://user-images.githubusercontent.com/47867495/53299237-e8b45580-3837-11e9-9006-d04562870720.png)

However, I can't seem to get any meaningful **Cluster Info** when clicking on the lines. If I click on the line "0000 Basic (22)" for instance, it gives me this:

![image](https://user-images.githubusercontent.com/47867495/53299275-392bb300-3838-11e9-9949-c75bdcad28ea.png)

But there seems to be _some_ info on a couple of lines. Here's from "0500 IASZone (5)":
![image](https://user-images.githubusercontent.com/47867495/53299310-8e67c480-3838-11e9-9c95-45b9ae511914.png)

And from "0019 OTAU (8)":
![image](https://user-images.githubusercontent.com/47867495/53299329-b525fb00-3838-11e9-99c1-6b6036f94d23.png)

Does this help?
Is there anything else I can do or information somewhere I can provide?
Or is this a lost case as it still seems impossible to get meaningful cluster information?

**Comment by Ronoon** [comment]
Well that should help so @manup  can look for a fault.
Got a similar problem and had to reset the sensor more times to default and then integrating it once again. So maybe you should try it once again to get a better result. 
In my case it was helpful and i got it once again to responding.

**Comment by manup** [comment]
We still need the attribute data from the Basic cluster like model identifier.

>However, I can't seem to get any meaningful Cluster Info when clicking on the lines. If I click on the line "0000 Basic (22)" for instance, it gives me this:

If this happens please restart deCONZ and try again, there seems to be a bug where the cluster widget is occasionally incomplete.

**Comment by jukmanty** [comment]
I have similar device:

![develco_MOSZB-130_ha_22_basic_cluster_0000](https://user-images.githubusercontent.com/553107/66703604-0f6ac200-ed1d-11e9-966b-12221351fce0.png)

Link to the technical manual of the device:

https://www.develcoproducts.com/media/4314/moszb-130-140-technical-manual-motion-sensor.pdf

**Comment by jukmanty** [comment]
I managed to connect MOSZB130 sensor using V2_05_73 beta version: https://github.com/dresden-elektronik/deconz-rest-plugin/issues/2011#issuecomment-590077023

**Comment by PrimaryInc** [comment]
Is it possible to reopen this thread or should I create a new "issue"/"request"

**Comment by Mimiix** [comment]
@PrimaryInc can you create a new one and add this issue as a reference?

**Comment by PrimaryInc** [comment]
> 
> 
> @PrimaryInc can you create a new one and add this issue as a reference?

Yes, will do :)
